### PR TITLE
feedClaim

### PR DESCRIPTION
**Pull request by sahdasamier**

### CODE REVIEW

**Review by gvelez17 — APPROVED** [review]
hey since we have other required changes, lets also make the image just a thumbnail please not a large image.  Look at a twitter feed or at [greencheck.](https://greencheck.world/timeline) or at https://easscan.org/

The idea is a simple feed that has many lines in it

Ours should be slightly more user friendly than the easscan.org one

but the idea is showing the items line by line, not a big area for each one

We probably want more of the statement in there and the thumbnail, so a little more user friendly and detailed but not too much

Can you please paste a screenshot on the next iteration?  thanks

File: src/containers/Search/index.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import styles from './styles'
 import SearchIcon from '@mui/icons-material/Search'
 import { parseNode, parseNodes } from './graph.utils'
 import { TextField } from '@mui/material'
+import FeedClaim from '../../feedCLAIM'
```
Comment (gvelez17) [inline src/containers/Search/index.tsx:16]: please do not use upper case in file names, its better to have all filenames lower case please

**Review by gvelez17 — CHANGES_REQUESTED** [review]
actually wait there are things that should change, see comments

File: src/feedCLAIM/index.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            object: 'http://trustclaims.whatscookin.us/local/company/VEJA',
+          </Typography>
+        </CardContent>
+        <CardActions disableSpacing>
+          <ExpandMore expand={expanded} onClick={handleExpandClick} aria-expanded={expanded} aria-label='show more'>
+            <ExpandMoreIcon />
+          </ExpandMore>
+        </CardActions>
+        <Collapse in={expanded} timeout='auto' unmountOnExit>
+          <CardContent>
```
Comment (gvelez17) [inline src/feedCLAIM/index.tsx:75]: we should never have fixed text like this in actual code, only in test cases.  please remove this hardcoded data from the src area

File: src/feedCLAIM/index.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+          </CardContent>
+        </Collapse>
+      </Card>
+
+      <Card sx={{ maxWidth: 'fit', m: '20px', borderRadius: '4px', border: '2px solid #80B8BD' }}>
+        <CardMedia component='img' height='194' src={polygon3} />
+        <CardContent>
+          <Typography sx={{ top: '10px', padding: '10px 0 0 10px' }}>
+            subject: 'https://www.bcorporation.net/',
+          </Typography>
```
Comment (gvelez17) [inline src/feedCLAIM/index.tsx:98]: remove all hardcoded text please.  these should be loaded dynamically

File: src/feedCLAIM/index.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        <CardActions disableSpacing>
+          <ExpandMore expand={expanded} onClick={handleExpandClick} aria-expanded={expanded} aria-label='show more'>
+            <ExpandMoreIcon />
+          </ExpandMore>
+        </CardActions>
+        <Collapse in={expanded} timeout='auto' unmountOnExit>
+          <CardContent>
+            <Typography sx={{ padding: '10px 0 0 10px' }}>
+              Statement: 'The VEJA project creates a positive chain. Sneakers are made differently using organic,
+              agroecological and fairtrade cotton to make the sneakers canvas, wild rubber from the Amazonian Forest for
```
Comment (gvelez17) [inline src/feedCLAIM/index.tsx:113]: in future remember to make it dynamic, there should never be a merge to master with any hardcoded text except in tests

Comment (sahdasamier) [reply src/feedCLAIM/index.tsx:113]: okay, i understand ,I removed all of that hard code and make the card empty, and the next task , will be dynamic, like our plan

File: src/containers/Search/index.tsx (hunk 1 above)
Comment (sahdasamier) [reply src/containers/Search/index.tsx:16]: no, a file called index , FeedClaim is a functional component @gvelez17

File: src/containers/Search/index.tsx [hunk 5]
```diff
@@ -209,19 +210,22 @@ const Search = (homeProps: IHomeProps) => {
...
-        selectedClaim={selectedClaim}
-        setLoading={setLoading}
-        setSnackbarMessage={setSnackbarMessage}
-        toggleSnackbar={toggleSnackbar}
-      />
-
-      <Box ref={ref} sx={styles.cy} />
-    </Container>
+    <Box>
+      {search ? null : <FeedClaim />}
```
Comment (ArnobChowdhury) [inline src/containers/Search/index.tsx:214]: Consider writing this line like below: 
`{!search && <FeedClaim />}`

Comment (sahdasamier) [reply src/containers/Search/index.tsx]: i made it @ArnobChowdhury

**Comment by gvelez17** [comment]
hey also do we need [src/assets/feed.jpg](https://github.com/Whats-Cookin/trust_claim/pull/141/files#diff-090014961acad25a1d2a2bc3b7595345160b8e4bd83bf04aacb12c2f57b5f877) ?  this looks like a test image right? lets not put it in assets unless we want it in the real live deployment

**Pull request closed without merging**